### PR TITLE
Tooltip tweaks

### PR DESCRIPTION
**Pull request by jimrandomh**
* In the tooltip when you hover over a PostsItem title, if the post is in multiple categories (eg Curated+Alignment Forum), show them both.
 * Add a sequence count to the user tooltip (uses the book icon).
 * When you hover over the date on a PostsPage for a curated post, show both the post date and the curation date.
 * When you hover over the frontpage/personal icon on PostsPage, add a title line.

### CODE REVIEW

File: packages/lesswrong/components/users/UsersNameDisplay.jsx [hunk 1]
```diff
@@ -24,6 +26,11 @@ const styles = theme => ({
     top: 2,
     color: "rgba(255,255,255,.8)"
   },
+  bookIcon: {
+    width: "1rem",
+    height: "1rem",
```
Comment (Raemon) [inline packages/lesswrong/components/users/UsersNameDisplay.jsx]: why does this have width/height when it's merged with icon via classNames?

File: packages/lesswrong/components/posts/PostsPage/PostsPageDate.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+  },
+});
+
+const PostsPageDate = ({ post, hasMajorRevision, classes }) => {
+  const { FormatDate, PostsRevisionSelector } = Components;
+  
+  const tooltip = (<div>
+    <div>Posted on <ExpandedDate date={post.postedAt}/></div>
+
```
Comment (Raemon) [inline packages/lesswrong/components/posts/PostsPage/PostsPageDate.jsx]: pretty sure html will remove this whitespace, but not quite sure why it's there in the first place

Comment (jimrandomh) [reply packages/lesswrong/components/posts/PostsPage/PostsPageDate.jsx]: Right, this blank line doesn't do anything. It's an arbitrary code-style thing.

File: packages/lesswrong/components/users/UsersNameDisplay.jsx (hunk 1 above)
Comment (jimrandomh) [reply packages/lesswrong/components/users/UsersNameDisplay.jsx]: They're redundant, I'll take them out.

**Comment by jimrandomh** [comment]
To merge this I'd have to use my admin-override, which I'd rather not. @Raemon?

**Pull request merged**